Pcap++/header/PcapFilter.h

```diff
@@ -484,113 +558,135 @@ namespace pcpp
 		 * Set the port
 		 * @param[in] port The port to create the filter with
 		 */
-		void setPort(uint16_t port) { portToString(port); }
+		void setPort(uint16_t port)
+		{
+			portToString(port);
+		}
 	};
 
-
-
 	/**
 	 * @class PortRangeFilter
-	 * A class for filtering TCP or UDP port ranges, meaning match only packets which port is within this range, for example: "src portrange 1000-2000"
-	 * will match only TCP or UDP traffic which source port is in the range of 1000 - 2000<BR>
+	 * A class for filtering TCP or UDP port ranges, meaning match only packets which port is within this range, for
+	 * example: "src portrange 1000-2000" will match only TCP or UDP traffic which source port is in the range of 1000 -
+	 * 2000
+	 *
 	 * For deeper understanding of the filter concept please refer to PcapFilter.h
 	 */
 	class PortRangeFilter : public IFilterWithDirection
 	{
 	private:
 		uint16_t m_FromPort;
 		uint16_t m_ToPort;
+
 	public:
 		/**
 		 * A constructor that gets the port range the the direction and creates the filter with them
 		 * @param[in] fromPort The lower end of the port range
 		 * @param[in] toPort The higher end of the port range
 		 * @param[in] dir The port range direction to filter (source or destination)
 		 */
-		PortRangeFilter(uint16_t fromPort, uint16_t toPort, Direction dir) : IFilterWithDirection(dir), m_FromPort(fromPort), m_ToPort(toPort) {}
+		PortRangeFilter(uint16_t fromPort, uint16_t toPort, Direction dir)
+		    : IFilterWithDirection(dir), m_FromPort(fromPort), m_ToPort(toPort)
+		{}
 
 		void parseToString(std::string& result) override;
 
 		/**
 		 * Set the lower end of the port range
 		 * @param[in] fromPort The lower end of the port range
 		 */
-		void setFromPort(uint16_t fromPort) { m_FromPort = fromPort; }
+		void setFromPort(uint16_t fromPort)
+		{
+			m_FromPort = fromPort;
+		}
 
 		/**
 		 * Set the higher end of the port range
 		 * @param[in] toPort The higher end of the port range
 		 */
-		void setToPort(uint16_t toPort) { m_ToPort = toPort; }
+		void setToPort(uint16_t toPort)
+		{
+			m_ToPort = toPort;
+		}
 	};
 
-
-
 	/**
 	 * @class MacAddressFilter
-	 * A class for filtering Ethernet traffic by MAC addresses, for example: "ether src 12:34:56:78:90:12" or "ether dst "10:29:38:47:56:10:29"<BR>
+	 * A class for filtering Ethernet traffic by MAC addresses, for example: "ether src 12:34:56:78:90:12" or "ether dst
+	 * "10:29:38:47:56:10:29"
```

Comment:
not directly related to this PR, but there is an extra `"` character in `"ether dst "10:29:38:47:56:10:29"`